app/models/round.rb

```diff
@@ -70,8 +70,8 @@ def find_process_date
 
   def self.generate_target_data
     wf = WeatherFacade.new
-    data = {:error=>{:code=>1006, :message=>"No matching location found."}}.to_json
-    while JSON.parse(data, symbolize_names: true)[:error]
+    data = {:error=>{:code=>1006, :message=>"No matching location found."}}
+    while data[:error]
```

Comment:
I feel the same issue with Parsing all the time, but I am glad that this was able to be updated by removing the JSON parse calls.